### PR TITLE
Add feedback buttons

### PR DESCRIPTION
**Pull request by seabo**

### CODE REVIEW

File: app/api/feedback/route.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Humanloop, ChatMessage } from "humanloop";
+
+if (!process.env.HUMANLOOP_API_KEY) {
+  throw Error(
```
Comment (harry-humanloop) [inline app/api/feedback/route.ts:4]: doesn't thins need to be `throw new Error()`? (idk what the difference is or what's right, just pointing a difference between what is here and what comes to my mind.

File: app/api/feedback/route.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const humanloop = new Humanloop({
+  apiKey: process.env.HUMANLOOP_API_KEY,
+});
+
+interface FeedbackRequest {
+  id: string;
+  value: string;
+}
+
+export async function POST(req: Request): Promise<Response> {
```
Comment (harry-humanloop) [inline app/api/feedback/route.ts:18]: https://nextjs.org/docs/pages/building-your-application/routing/api-routes

this doesn't seems to follow this pattern. any reason why?

```js
export default function handler(req, res) {
  res.status(200).json({ name: 'John Doe' })
}
```

File: app/api/feedback/route.ts (hunk 1 above)
Comment (seabo) [reply app/api/feedback/route.ts:4]: Yes, I think you're right. I believe they both technically work but you should use `new`

File: app/api/feedback/route.ts (hunk 2 above)
Comment (seabo) [reply app/api/feedback/route.ts:18]: This project is using the latest version of Next which has a new system for this called [Route Handlers](https://nextjs.org/docs/app/building-your-application/routing/router-handlers)

**Pull request merged**